src/App.tsx

```diff
@@ -8,11 +9,34 @@ const tabs: Tab[] = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+export type State = {
+  selectedTab: Tab,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedTab: tabs[0],
+  };
+
+  handleSelect = (tab: Tab) => {
```

Comment:
This function doesn't handle select element, choose more appropriate name.
```suggestion
  setSelectedTab = (tab: Tab) => {
```